packages/flutter/lib/src/material/expansion_tile.dart

```diff
@@ -256,29 +256,13 @@ class ExpansionTile extends StatefulWidget {
   ///   [ExpansionTileThemeData].
   final Color? collapsedTextColor;
 
-  /// The color of the tile's vertical border when the sublist is expanded.
-  ///
-  /// Used to override to the [Theme.dividerColor].
-  ///
-  /// If this property is null then [ExpansionTileThemeData.borderColor] is used. If that
-  /// is also null then the value of [Theme.dividerColor] is used.
-  ///
-  /// See also:
-  ///
-  /// * [ExpansionTileTheme.of], which returns the nearest [ExpansionTileTheme]'s
-  ///   [ExpansionTileThemeData].
-  final Color? borderColor;
+  /// Tile's border when the sublist is expanded.
+  /// If this property is null then a [Border] with vertical direction sides default to Color [Theme.divider] is used
+  final Border? border;
```

Comment:
Maybe ShapeBorder would be more flexible?

Reply:
I'll look into it

Reply:
ShapeBorderTween is a material.dart package dependent. Can I import it?, wouldn't that be a problem?